app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class MoviesController < ApplicationController
+  def index
+    @movies = Movie.all.map do |movie|
+      movie_params(movie)
+    end
+    render json: @movies
+  end
+
+  def show
+    movie = Movie.find(params[:id])
+    @transformed_movie = movie_params(movie)
+    render json: @transformed_movie
+  end
+
+  def new
+    Movie.create(movie_params)
```

Comment:
Does this create action work? Your `movie_params` function defined below  takes as parameter `movie`, which you are just trying to create.

Reply:
@agusiol has right, it won't work. Please look at this section of the guide to get to know how to set parameters for create https://edgeguides.rubyonrails.org/action_controller_overview.html#strong-parameters